design/features/confirmation-email-for-form-fillers/version-1.md

```diff
@@ -0,0 +1,217 @@
+> 
+> Thank you.  
+>
+> You cannot reply to this email. Use the contact details below if you need help with your form.  
+>
+> ### What happens next  
+> [Content will be whatever the form creator has specified on the 'Form submitted' page]  
+>
+> ### Contact details  
+> [Content will be whatever the form creator has specified on the 'Provide contact details for support' page - email, phone or online contact link]  
+
+
+#### Email examples
+
+The emails will include:
+
+- name of the form that the form filler has completed  
+- date and time the form filler submitted their form  
+- “Thank you” - this can add value to people completing forms, showing appreciation for their time, even if it’s a mandatory form    
+- information telling the form filler that they cannot reply to this email and referring them to the contact details provided later in the email
+- information about ‘What happens next’ which the form creator has added to their form's confirmation page  
+- contact information that the form creator has added to support their users  
+  
+![Access to work plus referral confirmation email example. Screenshot](https://github.com/alphagov/forms/assets/35372982/cf4d3bfd-b0d5-41b8-a8fc-281431861eee)  
+*This email shows an outline of the content and format that a typical email confirming that a submission's been sent might look like. This example shows information DWP might use for one of their forms.*
+
+![Apply for a county parish holding confirmation email example. Screenshot](https://github.com/alphagov/forms/assets/35372982/a884ec01-d0c8-4509-a6d1-09881530adf0)  
+*This email shows an outline of the content and format that a typical email confirming that a submission's been sent might look like. This example shows information Defra might use for one of their forms.*
+
+<br>
+
+## Design for form creators
+
+Giving form fillers control over whether or not they receive an email confirmation meant we needed to tell form creators about any information they provide which will be included in this email - and to make sure we do this at the appropriate stage of their form-creation journey. We decided that this would be at the point form creators add or edit their content for the "Information about what happens next" and "Contact details for support" pages. 
+
+### Form submitted page
+
+![Form submitted page. Screenshot](https://github.com/alphagov/forms/assets/35372982/24eabfbc-f961-43ea-8b4a-cc764af65a11)  
+
+We’ve added new content to this screen, just above the textarea input:
+
+> This content will also be included in an email confirming that a form's been submitted - if people choose to receive this. The email will include the date and time of submission, and contact details for support. It will not contain a copy of someone's answers.
+
+### Provide contact details for support
+
+![Provide contact details for support. Screenshot](https://github.com/alphagov/forms/assets/35372982/06fa052d-2cfd-4d31-aaea-1081ce02e187)  
```

Comment:
This screenshot is showing the 'Check your answers before submitting your form' page with the 'yes' radio expanded - rather than the 'Provide contact details for support' page

Reply:
Good spot. Fixed